packages/odyssey/src/scss/components/_table.scss

```diff
@@ -45,8 +45,8 @@
 
   thead {
     th {
-      border-top: 1px solid $base-border-color;
```

Comment:
is there a particular reason for the nested element selectors here? I'd love to see flatter specificity with classes!

Reply:
We've gone back and forth on this a bit. IMO there's very little good rationale for avoiding semantic markup when possible, and the nesting here is requisite to a properly formatted `table`. 

For non-html-spec elements, I absolutely agree that we should avoid nesting. (If I had my way, the entire Table component would be an unclassed, default `table`, but our products are too brittle for that right now.)

Reply:
@edburyenegren-okta thanks for this response. I'm not suggesting avoiding semantic markup. I'm very much in favor because of all the built in accessibility benefits and the easier development it provides.

Tidy CSS should be an orthogonal concern to semantic markup. We can have both.

If our selectors are always coupled closely to the structure of the DOM, or perhaps written in a way that provides different styles based on an awareness of their content, or perhaps relying on ever increasing specificity to apply the styles we want for a specific context than we will be cursing ourselves in the future. This will become difficult to maintain and understand.